doc/examples/compositing/contactSheet.gfr

```diff
@@ -400,214 +400,234 @@ __children["Dot7"]["in"].setInput( __children["Dot6"]["out"] )
 __children["Expression1"]["__engine"].setValue( 'python' )
-__children["Expression1"]["__expression"].setValue( 'parent["__out"]["p0"] = context.get( "contactSheet:column", 0 ) * parent["Options"]["user"] ["baseResolution"]' )
+__children["Expression1"]["__expression"].setValue( 'parent["__out"]["p0"] = context.get( "contactSheet:column", 0 ) * parent["Settings"]["user"]["baseResolution"]' )
 __children["Expression2"]["__engine"].setValue( 'python' )
-__children["Expression2"]["__expression"].setValue( 'parent["__out"]["p0"] = context.get( "contactSheet:row", 0 ) * parent["Options"]["user"] ["baseResolution"]' )
-__children["Wedge_Scene_and_Shaders"]["Expression"]["__engine"].setValue( 'OSL' )
-__children["Wedge_Scene_and_Shaders"]["Expression"]["__expression"].setValue( 'parent.__out.p0 = context("wedge:metallic", 0.0);\nparent.__out.p1 = context("wedge:roughness", 0.0);' )
+__children["Expression2"]["__expression"].setValue( 'parent["__out"]["p0"] = context.get( "contactSheet:row", 0 ) * parent["Settings"]["user"]["baseResolution"]' )
+__children["WedgeSceneAndShaders"]["Expression"]["__engine"].setValue( 'OSL' )
+__children["WedgeSceneAndShaders"]["Expression"]["__expression"].setValue( 'parent.__out.p0 = context("wedge:metallic", 0.0);\nparent.__out.p1 = context("wedge:roughness", 0.0);' )
 __children["Expression"]["__engine"].setValue( 'python' )
-__children["Expression"]["__expression"].setValue( 'parent["__out"]["p0"] = parent["Options"]["user"] ["steps"]' )
+__children["Expression"]["__expression"].setValue( 'parent["__out"]["p0"] = parent["Settings"]["user"] ["steps"]' )
 __children["Expression3"]["__engine"].setValue( 'python' )
-__children["Expression3"]["__expression"].setValue( 'parent["__out"]["p0"] = parent["Options"]["user"] ["steps"]' )
+__children["Expression3"]["__expression"].setValue( 'parent["__out"]["p0"] = parent["Settings"]["user"] ["steps"]' )
 __children["Expression5"]["__engine"].setValue( 'python' )
-__children["Expression5"]["__expression"].setValue( 'size = parent["Options"]["user"] ["steps"] * parent["Options"]["user"] ["baseResolution"]\nparent["__out"]["p0"] = size\nparent["__out"]["p1"] = size' )
+__children["Expression5"]["__expression"].setValue( 'size = parent["Settings"]["user"] ["steps"] * parent["Settings"]["user"]["baseResolution"]\nparent["__out"]["p0"] = size\nparent["__out"]["p1"] = size' )
+__children["Backdrop14"] = Gaffer.Backdrop( "Backdrop14" )
+parent.addChild( __children["Backdrop14"] )
+__children["Backdrop14"].addChild( Gaffer.Box2fPlug( "__uiBound", defaultValue = imath.Box2f( imath.V2f( -10, -10 ), imath.V2f( 10, 10 ) ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Backdrop14"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Backdrop14"]["title"].setValue( 'Tip: Connect to Next' )
```

Comment:
Do you think we could tone down this backdrop color too - its quite saturated compared to others?

Reply:
2f861a2cba670582a063d0cb98a09bf9f6764cd5